pyccel/ast/basic.py

```diff
@@ -1,3 +1,12 @@
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+
+"""
+basic module docstring.
+"""
```

Comment:
As we have already said we can temporarily de-activate the codacy warnings about missing module docstrings when we merge this branch. The worst thing you can do is provide a placeholder docstring as you have done here as it hides the fact that the docstring is missing.
You have 3 options:
1. Do not provide a docstring, we will make sure we can still merge
2. Try and understand what the module does and provide a basic docstring
3. Try and identify someone who knows what's in the file and ask them to provide a basic docstring